src/test/java/net/kaczmarzyk/E2eTestBase.java

```diff
@@ -140,7 +139,6 @@ public void initialize() {
                 .lastSeen(Timestamp.valueOf(LocalDateTime.of(2022, 12, 6, 15, 6, 2, 0)))
                 .build(em);
 
-        TestLogAppender.clearInterceptedLogs();
```

Comment:
I removed this interceptor as it was replaced with HibernateStatementInspector (?)